components/FeatureHighlight/src/MDCFeatureHighlightViewController.h

```diff
@@ -1,5 +1,5 @@
 /*
- Copyright 2015-present the Material Components for iOS authors. All Rights Reserved.
+ Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
```

Comment:
Please don't change copyright years—the first year is the year when the file was created.